### DISCUSSION TITLE
how to set SMTP Provider Settings using supabase createClient

### DISCUSSION THREAD

**Discussion by webdeveloper20213** [supabase/supabase#17140] [Questions] [answered]
how to set **SMTP Provider Settings** using supabase createClient like how to set **HOST & PORT name** or **enable/disable** using client side.
Is there any method available or any apis or only one way using dashboard to set this creds?

![image](https://github.com/supabase/supabase/assets/133633860/1a9beb54-d78c-47f7-b1de-19b9b3d494b5)

**Comment by silentworks** [comment, marked as answer]
This can be done only via the dashboard.

**Comment by webdeveloper20213** [comment]
Thanks for a quick response @silentworks 

**Auth Providers** also using dashboard right?

![image](https://github.com/supabase/supabase/assets/133633860/4bba2361-1260-42cb-8ff2-04ea3d404297)

**Reply by silentworks** [reply]
Yes that's correct too.
